### ISSUE TITLE
Display attributes on mouse over 

### ISSUE THREAD

**Issue by suthers** [gephi/gephi#459]
We're interpreting a large graph by looking at node attributes in the clusters. The #1 feature that would help a lot and should be easy to add: Set a toggle to turn on mouse over to display attributes in tool-tip format. It could display just those attributes that are selected for display in the Data Laboratory. This would save the many round trips back and forth between Overview and Data Laboratory as we are trying to figure out what nodes participate in a given community.  Thanks so much; we are already getting considerable work done with this software. s

**Comment by micahstubbs** [comment]
+1 I'd enjoy this feature as well

**Comment by Tafelberger** [comment]
+1 

BadgeGeo has such a feature, too:
When you hover over or click on a node, it will show you selected XML information from your data file.
This also helps you in categorising nodes, not only the weight of their edges.

Here is just an example of that:
https://intothelooneybin.files.wordpress.com/2014/04/screen-shot-2014-04-21-at-14-49-04.png?w=219&h=98

PS: this is such an old thread, I don't even now if it is still monitored.

**Comment by jbliesener** [comment]
The issue looks still active and open, +1 for an (optional) tooltip with (selectable?) node attributes. Configuration could be done through the "Graph options" panel, with a "Tooltip" tab.

**Comment by irgb** [comment]
+1 I'd enjoy this feature as well

**Comment by chansh518** [comment]
Need this feature very much!

**Comment by chansh518** [comment]
I just found the hover function is already available. In Graph window, expand the panel which configures graph formats. There are three tabs: Global, Edges, Labels. In Labels, there is a selection: Hide non-selected. If you click this selection, then all labels will be hidden. Only the one you clicked will show its label. You can configure what the label is. Check all the attributes you want as the labels.

**Comment by gbolcer** [comment]
Confirming.  It took a little hunting to find the actual configuration panel expansion button on the lower right of the graph pane, but it works perfectly. 

https://www.youtube.com/watch?v=xJrWW8WgEzE&feature=youtu.be

**Comment by eduramiba** [comment]
We should make all these options more obvious

**Comment by gbolcer** [comment]
I think part of my issue was that I have it in fullscreen mode on a 3840x2160 monitor.  I have Text scaling turned on and set to 175% (win10 x64), but Gephi doesn't seem to use those settings.    Thanks!

**Comment by eduramiba** [comment]
@gbolcer Check #1702

**Comment by worknet-bienew** [comment]
It will be great to have a tool-tip feature for the nodes attributes. Similar to NodeXL.

**Comment by SwethaMagesh** [comment]
![image](https://user-images.githubusercontent.com/43994542/112629012-bd0e5a00-8e59-11eb-9d06-e0e1d7c66456.png)
Hide non selected works fine. This doesn't need plugin and serves the purpose

**Comment by SwethaMagesh** [comment]
You can select whichever node attribute you need to view too.
![image](https://user-images.githubusercontent.com/43994542/112629154-ef1fbc00-8e59-11eb-94ee-afa8f863b0dd.png)

**Comment by lipchev** [comment]
Yes, but this is not the same as having the _short_ attribute visible as per the _Label text settings_, while the _longer_ one appears on mouse over.

**Comment by harrystuart** [comment]
Have there been any developments on a tooltip? This would be the single most helpful feature for my use-case and I imagine, many others. Cheers

**Comment by Trevoke** [comment]
Adding my voice to this 25-year-old issue! We're looking at a very large graph that has a lot of nodes and edges with different properties, so pre-configuring what we're going to see is a daunting task. Some docked window on the side of the screen that would show the node/edge properties (similar, in a way, to what neo4j frontend allows) would be fantastic. 

Hover is also nice but I like moving my mouse :) 

Massive bonus - if we can display separately all the properties that exist but have a NULL value.
